mdflynn/whats-cookin
# Feature/buttons on modal

## What is the change?
- This adds the favorite a recipe button and add to grocery list button to the modal view.
- This adds event listeners for the new buttons.
- Styles modal.
- Adds ingredient measurements to modal.

## What does it fix?
This refactors the displayHandler method to adhere to SRP roles.

## Is this a fix or a feature?
Both a fix and a feature.

## Where should the reviewer start?
Lines 9-175 in the sripts.js file.

## How should this be tested?
Run ```open src/index.html``` in your terminal to see new site functionality and design.
